README.md

```diff
@@ -9,6 +9,401 @@
+
+In React, we can dynamically render child components by pulling them off of the `children` property on our components props. Let's update our `MoviesPage` so that it renders it's child components underneath the `MoviesList`
+
+```javascript
+// src/containers/MoviesPage.js
+import React, { Component } from 'react';
+...
+
+class MoviesPage extends Component {
+...
+
+  render(){
+    return(
+      <div>
+        <MoviesList movies={ this.props.movies } />
+        { this.props.children }
+      </div>)
+  }
+}
+
+...
+
+export default connect(mapStateToProps, mapDispatchToProps)(MoviePage);
+```
+
+Now, any child components provided by ReactRouter will be rendered there. Awesome! Refresh again - now we see our `MoviesShow` component displayed at our dynamic route.
+
+
+### Dynamically finding the show
+
+We've successfully created out nested route. Next, let's wire up our `MoviesShow` component to dynamically render the info about the movie based on the URL. The steps to do so will be as follows:
+
+1. Connect our MoviesShow component to the store so that it knows about the list of movies.
+2. Find the movie where the movie's id matches the `:id` param of our route.
+3. Make that movie available to the component via `props`.
+
+First, let's import `connect` and use our `mapStateToProps` function to let our `MoviesShow` component know about changes to the store.
+
+```javascript
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+
+class MoviesShow extends Component {
+
+  render(){
+    return (
+      <div>
+        Movies Show Component
+      </div>
+    )
+  }
+}
+
+function mapStateToProps(state){
+
+}
+export default connect(mapStateToProps)(MoviesShow);
+```
+
+Now, in `mapStateToProps`, we'd like to access the `:id` supplied to us via the URL. We need to understand two things for this to work.
+
+1. `mapStateToProps` takes a second argument of props that were passed directly to the component. We usually refer to these as `ownProps`
+2. React Router will supply any dynamic pieces of the URL to the component via an object called `routeParams`
+
+This means that we can access the `:id` from the URL via `routeParams` on our `ownProps`
+
+```javascript
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+
+class MoviesShow extends Component {
+
+  render(){
+    return (
+      <div>
+        Movies Show Component
+      </div>
+    )
+  }
+}
+
+function mapStateToProps(state, ownProps){
+  ownProps.routeParams.id // this will return the dynamic portion of the url.
+}
+
+export default connect(mapStateToProps)(MoviesShow);
+```
+
+Note that we have a property called `id` because of the way we defined our route. If we defined our dynamic portion to be `/movies/:dog`, we'd have a `dog` property in our `routeParams`.
```

Comment:
I'm glad this point was included! I know i've taken for granted that students understand this idea in Rails